solutions/cpython-tests/Dockerfile

```diff
@@ -1,10 +1,19 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 ARG CPYTHON_TAG=v3.8.11
 WORKDIR /
 
-RUN apt update && apt install -y build-essential libssl-dev zlib1g-dev libncurses5-dev libncursesw5-dev \
-        libreadline-dev libsqlite3-dev libgdbm-dev libdb5.3-dev libbz2-dev \
-        libexpat1-dev liblzma-dev libffi-dev git uuid-dev lsb-core
+RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y build-essential libssl-dev libncurses5-dev libncursesw5-dev \
+libreadline-dev libsqlite3-dev libgdbm-dev libdb5.3-dev libbz2-dev \
+libexpat1-dev liblzma-dev libffi-dev git uuid-dev lsb-core wget
+
+# Workaround broken crc32_z symbol on zlib 1.2.11 for Ubuntu 20.04
```

Comment:
Hmm thats seems puzzling. The symbol type is different in the above output - FUNC(18.04) vs IFUNC(20.4). It might be our musl loader doesn't support IFUNC symbols.
cc: @anakrish

Reply:
This is the traceback:
```
test test_exceptions crashed -- Traceback (most recent call last):
  File "/cpython/Lib/test/libregrtest/runtest.py", line 335, in _runtest_inner
    refleak = _runtest_inner2(ns, test_name)
  File "/cpython/Lib/test/libregrtest/runtest.py", line 280, in _runtest_inner2
    the_module = importlib.import_module(abstest)
  File "/cpython/Lib/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/cpython/Lib/test/test_exceptions.py", line 13, in <module>
    from test.support import (captured_stderr, check_impl_detail,
  File "/cpython/Lib/test/support/script_helper.py", line 11, in <module>
    import zipfile
  File "/cpython/Lib/zipfile.py", line 6, in <module>
    import binascii
ImportError: Error relocating /usr/lib/x86_64-linux-gnu/libz.so.1: crc32_z: symbol not found

test_exceptions failed (uncaught exception)

== Tests result: FAILURE ==

1 test failed:
    test_exceptions
 ```